pom.xml

```diff
@@ -21,7 +21,7 @@
     <groupId>bio.overture</groupId>
     <artifactId>song</artifactId>
     <packaging>pom</packaging>
-    <version>3.0.1-SNAPSHOT</version>
+    <version>3.0.2-SNAPSHOT</version>
```

Comment:
I thought we said we were only going to increment version numbers for a release to the master branch?

Reply:
yes you are right, i completely forgot!